package.json

```diff
@@ -15,10 +15,10 @@
         "lint:js": "eslint . --ext .js",
         "lint:json": "jshint -e .json www editor/blocks editor/extensions editor/interface editor/paint-editor",
         "lint": "npm run lint:js && npm run lint:json",
-        "pull:interface": "babel-node scripts/tx-pull-translations scratch-editor interface ./editor/interface/",
```

Comment:
I'm curious about this rename: it seems like `tx-pull-editor` implies that it's only being used for `scratch-editor` but I see that it's still being used for other projects. Is the idea that these are all "editor-ish" projects in some way?

Reply:
From what I understand, yeah - could certainly rename it, although the rename does fit with the way the repo is structured (`pull:www` puts stuff in the www folder, `pull:editor` puts stuff in the editor folder).

Reply:
Ah, that makes sense. Thanks!